src/it/java/com/google/checkstyle/test/chapter5naming/rule526parameternames/ParameterNameTest.java

```diff
@@ -63,6 +63,14 @@ public void parameterNameTest() throws Exception {
             "26:21: " + getCheckMessage(checkConfig.getMessages(), MSG_KEY, "bB", format),
             "49:22: " + getCheckMessage(checkConfig.getMessages(), MSG_KEY, "llll_llll", format),
             "50:21: " + getCheckMessage(checkConfig.getMessages(), MSG_KEY, "bB", format),
+            "61:23: " + getCheckMessage(checkConfig.getMessages(), MSG_KEY, "p", format),
+            // To be removed after issue #3381 is closed
+            "65:24: " + getCheckMessage(checkConfig.getMessages(), MSG_KEY, "p", format),
+            // To be removed after issue #3381 is closed
+            "72:31: " + getCheckMessage(checkConfig.getMessages(), MSG_KEY, "p", format),
+            "77:41: " + getCheckMessage(checkConfig.getMessages(), MSG_KEY, "p", format),
+            // To be removed after issue #3381 is closed
```

Comment:
please remove such comments and place them to github issue description or comment.